### PR TITLE
Add change cluster folder option for some DBs

### PR DESCRIPTION
**Pull request by Cockamamie**
MongoDB, Elasticsearch, Kafka, ClickHouse, Redis

### CODE REVIEW

**Comment by Denchick** [comment]
we agreed to issue a grant to @Cockamamie so he would provide proof that MoveRequest works

**Comment by Cockamamie** [comment]
There's operation ids to prove
Mongo `c9qcom56hcqnlnamd7nf`
Redis: `rd4mov2og883uq2t5rsg`
Kafka: `dqhke4fofh5fvqve4ual`
ClickHouse: `c5m2kedre965j8fr01c5`

**Comment by Denchick** [comment]
I reviewed it very carefully and ran acceptance tests. No problems found, look great to me. Let's merge it?

**Review by Denchick — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,8 +22,11 @@ FEATURES:
 * added `shared_credentials_file` and `profile` provider properties
 * storage: support configuration of `storage_access_key`/`storage_secret_key` keys from shared credentials file
 * redis: support `maxmemory_percent` in `yandex_mdb_redis_cluster`
-* compute: support updating host_affinity_rules in the `yandex_compute_instance` resource.
 * storage: added `source_hash` property to `storage_object` resource
+* mongodb: changing `folder_id` attribute in `yandex_mdb_mongodb_cluster` moves MongoDB cluster to new folder
```
Comment (opportunity356) [inline CHANGELOG.md]: Let's move this changes to current release

**Comment by Apelsin234** [comment]
I guess we need tests covering this feature

**Comment by Cockamamie** [comment]
> I guess we need tests covering this feature

I've seen commit `6035fdb5a851f3b1b990d69fe851b858699457e5` with implementation of similar functionality for PostgreSQL and there was no tests.
Could you explain pls why we need tests?

File: CHANGELOG.md (hunk 1 above)
Comment (Cockamamie) [reply CHANGELOG.md]: done

**Review by opportunity356 — APPROVED** [review]

**Comment by Denchick** [comment]
> I guess we need tests covering this feature

Well, when we added this function for PostgreSQL, we didn't come up with an idea of how to do it. Inside one test you should use two test folders and it seems like too many actions for testing one specific feature.

Do you know an easy way to test it?

**Comment by Apelsin234** [comment]
I dunno why in postgres decided not to add tests. 
Imho, every functionality should be covered by tests. We want to migrate part of mongodb and redis for this month from sdk2 to terraform-plugin, and I am sure that the tests will not be superfluous.

P.s. You can add tests to next pr.

**Comment by Apelsin234** [comment]
You can add an additional folder in our cloud for it, we have already done this for the restore test.  And add a move to and from this folder to the existing test.

**Pull request merged**